Add documentation on Opcode Assertions method (#68)
lambdaclass/cairo-vm.go: README.md

    //TODO
```

*Step*

```go
    //TODO
```

*Decode instruction*

```go
    //TODO
```

*Run instruction*

```go
    //TODO
```

*Compute operands*

```go
    //TODO
```

*Opcode assertions*

> Once we have the instruction's operands to work with, we have to ensure the correctness of them. The first thing we need to differenciate is which type of instruction are we running, we do this by switching the `opcode`. 

Reviewer: > we do this by switching the `opcode`. 

This line is confusing, we could instead say, by looking at the instruction's opcode

Author: Ty!

> Once we have the instruction's operands to work with, we have to ensure the correctness of them. The first thing we need to differenciate is which type of instruction are we running, we do this by switching the `opcode`. 

Author: ```suggestion
Once we have the instruction's operands to work with, we have to ensure the correctness of them. The first thing we need to differentiate is which type of instruction are we running, we do this by looking at the instruction's opcode. 
```

> The posible options (for now) are: 

Reviewer: This is not correct, there are no other options waiting to be added, we could instead say The options we care about in this method/ The options we want to perform assertions on

Reviewer: Also we could refer to the as opcodes

Author: ty!

> The posible options (for now) are: 

Author: ```suggestion
The posible opcodes we want to perform assertions on are: 
```

	1. AssertEq instruction 
	2. Call instruction 

> In the first option, we need to ensure the result operand is not null (nil in this case) and also that the result operand is equal to the destiny operand. If any of those things fail, we throw an error. 

Author: ```suggestion
In the first option, we need to ensure the result operand is not null (nil in this case) and also that the result operand is equal to the dst operand. If any of those things fail, we throw an error. 
```

> On the other hand, the Call instruction, what we do first is define our return PC register, we do that adding the size of the instruction to the current PC. Then, we check our operand OP0 is equal to the return PC and our destiny operand is the same as the return FP register. If any of those things fail, we throw an error. 

Reviewer: ```suggestion
On the other hand, the Call instruction, what we do first is define our return PC register, we do that adding the size of the instruction to the current PC. Then, we check our operand OP0 is equal to the return PC and our destiny operand is the same as the return FP register. If any of those things fail, we throw an error. 
```
destiny operand doesn't sound quite right, we should use destination, or just dst instead

Author: I agree!

> On the other hand, the Call instruction, what we do first is define our return PC register, we do that adding the size of the instruction to the current PC. Then, we check our operand OP0 is equal to the return PC and our destiny operand is the same as the return FP register. If any of those things fail, we throw an error. 

Author: ```suggestion
On the other hand, the Call instruction, what we do first is define our return pc register, we do that adding the size of the instruction to the current pc. Then, we check our operand op0 is equal to the return pc and our dst operand is the same as the return fp register. If any of those things fail, we throw an error. 
```

> If this method returns nil, it means operands are correct and we are good to go!

Reviewer: ```suggestion
If this method returns a nil error, it means operands were computed correctly and we are good to go!
```
